### PR TITLE
feat: imported babel plugin

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Onboards our [Babel preset](https://github.com/Codecademy/babel-preset-codecademy) to client-modules with `yarn lerna import`.

There wasn't any particular reason to keep it separate from our ESLint, Webpack, etc. configs.

### PR Checklist

- ~[ ] Related to designs:~
- [x] Related to JIRA ticket: WEB-1272
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

https://github.com/yarnpkg/yarn/issues/7807 explains the `yarn.lock` recreate and https://github.com/jhipster/generator-jhipster/pull/12664 explains the enzyme version bump.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>babel-preset-codecademy@3.0.2-alpha.3b2386.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601c883583c31400b41bd86f--gamut-preview.netlify.app>https://601c883583c31400b41bd86f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601c883583c31400b41bd86f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>babel-preset-codecademy@3.0.2-alpha.252077.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601d3bc95f91d4fda47bccb7--gamut-preview.netlify.app>https://601d3bc95f91d4fda47bccb7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601d3bc95f91d4fda47bccb7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>babel-preset-codecademy@3.0.2-alpha.ce8b86.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601d48f349428ce098600b68--gamut-preview.netlify.app>https://601d48f349428ce098600b68--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601d48f349428ce098600b68>Deploy Logs</a></p>

File: packages/babel-preset-codecademy/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# babel-preset-codecademy
```
Comment (dreamwasp) [inline packages/babel-preset-codecademy/README.md:1]: 👏👏👏👏👏

**Review by dreamwasp — APPROVED** [review]

**Review by jakemhiller — APPROVED** [review]

**Review by codecaaron — APPROVED** [review]
Might want to just check resolutions with the monolith with the lock file regen, but otherwise, this is awesome

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>babel-preset-codecademy@3.0.2-alpha.d0bdf5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6025880987b0840e6fa91952--gamut-preview.netlify.app>https://6025880987b0840e6fa91952--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6025880987b0840e6fa91952>Deploy Logs</a></p>

**Pull request merged**